### PR TITLE
Update the Openshift connector plugin to latest version

### PR DESCRIPTION
**Pull request by vinokurig**
Signed-off-by: Igor Vinokur <ivinokur@redhat.com>

### What does this PR do?
Update the Openshift connector plugin to latest version

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1733

### CODE REVIEW

**Comment by vitaliy-guliy** [comment]
I suppose `Signed-off-by` should contain Red Hat email

**Review by ericwill — COMMENTED** [review]
I believe the dependencies in the sidecar need to be updated. Please also make this change upstream.

File: dependencies/che-plugin-registry/che-theia-plugins.yaml [hunk 1]
```diff
@@ -259,14 +259,14 @@ plugins:
...
-      revision: v0.1.5
+      revision: v0.2.7
     sidecar:
       image: "registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.10"
       name: vscode-openshift-connector
       memoryLimit: 1500Mi
       cpuLimit: 500m
       cpuRequest: 30m
-    extension: https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-openshift-tools/vscode-openshift-connector-0.1.5.vsix
+    extension: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v31f30bd/vscode-openshift-connector-v0.2.7.vsix
```
Comment (svor) [inline dependencies/che-plugin-registry/che-theia-plugins.yaml:269]: Shouldn't it be downloaded from https://download.jboss.org/jbosstools/vscode/stable/vscode-openshift-tools/ or https://open-vsx.org/api/redhat/vscode-openshift-connector/ ?

Comment (ericwill) [reply dependencies/che-plugin-registry/che-theia-plugins.yaml:269]: Nope, the link is correct. The extension was built from source and published as a GitHub release asset: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions

**Comment by vinokurig** [comment]
@ericwill 
> I believe the dependencies in the sidecar need to be updated. 

Could you please clarify what dependency is needed to be updated?

> Please also make this change upstream.

I think it depends on https://github.com/eclipse/che/issues/19361

**Comment by nickboldt** [comment]
Note that the Build PR check is failing because your fork cannot get access to secrets. 

So here's a replacement PR https://github.com/redhat-developer/codeready-workspaces/pull/542

**Comment by ericwill** [comment]
> @ericwill
> 
> > I believe the dependencies in the sidecar need to be updated.
> 
> Could you please clarify what dependency is needed to be updated?

In the upstream plugin repository, there is a list: https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/tools.json

**Pull request closed without merging**